Modules/Devices/src/Devices.Application/Identities/Commands/StartDeletionProcessAsOwner/Handler.cs

```diff
@@ -33,6 +38,8 @@ public async Task<StartDeletionProcessAsOwnerResponse> Handle(StartDeletionProce
 
         await _identitiesRepository.Update(identity, cancellationToken);
 
+        await _notificationSender.SendNotification(identity.Address, new DeletionProcessApprovedNotification(deletionProcess.GracePeriodEndsAt?.DaysUntilDate() ?? 0), cancellationToken);
```

Comment:
I don't think we should send a DeletionProcessApprovedNotification. The user doesn't know that the process is implicitly approved. That's an internal thing. The notification should just say something like "deletion process started". I'm open for better suggestions though